cluster/addons/cluster-monitoring/v1beta3/influxdb-grafana-controller.yaml

```diff
@@ -24,10 +24,10 @@ spec:
             - containerPort: 8086
               hostPort: 8086
         - name: grafana
-          image: gcr.io/google_containers/heapster_grafana:v0.6
+          image: gcr.io/google_containers/heapster_grafana:v0.7
           env: 
             - name: "INFLUXDB_EXTERNAL_URL"
-              value: '/api/v1beta1/proxy/services/monitoring-grafana/db/'
+              value: '/api/v1beta3/proxy/namespaces/system/services/monitoring-grafana/db/'
```

Comment:
Have you tested this change? If you were to make this change on the heapster repo [here](https://github.com/GoogleCloudPlatform/heapster/blob/master/deploy/kube-config/influxdb/v1beta3/influxdb-grafana-controller.json#L41), it will get tested automatically by CI!